ls1intum/Artemis
# Bugfix/Lazy loading the admin module on production/test server

### Checklist

### Motivation and Context



Issue from https://github.com/ls1intum/Artemis/pull/736.
While working locally, the admin module fails on lazy load on the server:
```
breadcrumbs.js:58 ERROR Error: Uncaught (in promise): ReferenceError: Cannot access 'vl' before initialization
ReferenceError: Cannot access 'vl' before initialization
```

The strange thing is that this was working when I tested the PR on the test server: https://github.com/ls1intum/Artemis/pull/736#discussion_r317500373

### Description


It seems like it was just the way the routes were imported.

### Steps for Testing


1. Log in to Artemis (on test server!)
2. Open area in admin module (e.g. User management)
3. Area should be loaded

### Screenshots
